packages/visx-wordcloud/src/useWordcloud.ts

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from 'react';
+import d3Cloud from 'd3-cloud';
+
+export interface BaseDatum {
```

Comment:
We could extract `BaseDatum` and `WordcloudConfig` types into `./types.ts` so this file can export `useWordcloud` as default export, making it consistent with `Wordcloud.tsx`.